src/app/layout.tsx

```diff
@@ -1,17 +1,26 @@
 import type { Metadata } from 'next'
-import { Inter } from 'next/font/google'
+import { Roboto } from 'next/font/google'
```

Comment:
I added global css with Roboto font applied to body, there is no need to add Roboto font here. Check my Footer PR, let me know if works for you this way.